File: IHIS/IHIS.Framework/IHIS.CloudConnector/Contracts/Models/Cpls/CPL0000Q00ResultListQuerySelect1ListItemInfo.cs
using System;

namespace IHIS.CloudConnector.Contracts.Models.Cpls
{
    [Serializable]
	public class CPL0000Q00ResultListQuerySelect1ListItemInfo
	{
		private String _serial;
		private String _kyunResult;
		private String _kyunCode;
		private String _micName;
		private String _micResult;

		public String Serial
		{
			get { return this._serial; }
			set { this._serial = value; }
		}

		public String KyunResult
		{
			get { return this._kyunResult; }
			set { this._kyunResult = value; }
		}

		public String KyunCode
		{
			get { return this._kyunCode; }
			set { this._kyunCode = value; }
		}

		public String MicName
		{
			get { return this._micName; }
			set { this._micName = value; }
		}

		public String MicResult
		{
			get { return this._micResult; }
			set { this._micResult = value; }
		}

		public CPL0000Q00ResultListQuerySelect1ListItemInfo() { }

		public CPL0000Q00ResultListQuerySelect1ListItemInfo(String serial, String kyunResult, String kyunCode, String micName, String micResult)
		{
			this._serial = serial;
			this._kyunResult = kyunResult;
			this._kyunCode = kyunCode;
			this._micName = micName;
			this._micResult = micResult;
		}

	}
}

File: MED-CLOUD/med-data/src/main/java/nta/med/data/model/ihis/cpls/CPL0000Q00ResultListQuerySelect1ListItemInfo.java
package nta.med.data.model.ihis.cpls;

public class CPL0000Q00ResultListQuerySelect1ListItemInfo {
	private Double serial;
    private String kyunResult;
    private String kyunCode;
    private String micName;
    private String micResult;
	public CPL0000Q00ResultListQuerySelect1ListItemInfo(Double serial,
			String kyunResult, String kyunCode, String micName, String micResult) {
		super();
		this.serial = serial;
		this.kyunResult = kyunResult;
		this.kyunCode = kyunCode;
		this.micName = micName;
		this.micResult = micResult;
	}
	public Double getSerial() {
		return serial;
	}
	public void setSerial(Double serial) {
		this.serial = serial;
	}
	public String getKyunResult() {
		return kyunResult;
	}
	public void setKyunResult(String kyunResult) {
		this.kyunResult = kyunResult;
	}
	public String getKyunCode() {
		return kyunCode;
	}
	public void setKyunCode(String kyunCode) {
		this.kyunCode = kyunCode;
	}
	public String getMicName() {
		return micName;
	}
	public void setMicName(String micName) {
		this.micName = micName;
	}
	public String getMicResult() {
		return micResult;
	}
	public void setMicResult(String micResult) {
		this.micResult = micResult;
	}
}
